### PR TITLE
An tabbed interface

### PR DESCRIPTION
**Pull request by AdolfoNava**

### CODE REVIEW

**Review by dylansarikas — APPROVED** [review]
Looks good!  I like that you made a user partial in the users views

**Review by CarlosCastanedaDev — APPROVED** [review]
Nice job creating the navbar partial!

You can now merge these changes back to main.

File: config/routes.rb
Comment (jrzricardo) [inline config/routes.rb:1]: new routes look solid.

File: app/views/users/show.html.erb
Comment (jrzricardo) [inline app/views/users/show.html.erb:1]: nice refactor.

File: app/views/users/liked.html.erb
Comment (jrzricardo) [inline app/views/users/liked.html.erb:1]: clean.

File: app/views/users/following.html.erb
Comment (jrzricardo) [inline app/views/users/following.html.erb:1]: looks like followers.

File: app/views/users/followers.html.erb
Comment (jrzricardo) [inline app/views/users/followers.html.erb:1]: looks like following.

File: app/views/users/_user.html.erb
Comment (jrzricardo) [inline app/views/users/_user.html.erb:1]: this was nice touch, way to reduce repetitions!

File: app/views/users/_navbar.html.erb
Comment (jrzricardo) [inline app/views/users/_navbar.html.erb:1]: solid partial.

File: app/views/shared/_navbar.html.erb
Comment (jrzricardo) [inline app/views/shared/_navbar.html.erb:1]: sweet change, now the user will recognize themselves.

File: app/controllers/users_controller.rb
Comment (jrzricardo) [inline app/controllers/users_controller.rb:1]: new methods look good.

**Review by jrzricardo — COMMENTED** [review]
wonderful code! I get the feeling you prefer to include "partial" when you render. I respect that choice :>

**Pull request merged**